extensions/common/proto_util_test.cc

```diff
@@ -46,7 +46,7 @@ constexpr absl::string_view node_metadata_json = R"###(
    "WORKLOAD_NAME":"test_workload",
    "OWNER":"test_owner",
    "NAME":"test_pod",
-   "APP_CONTAINERS": [ "test", "hello" ]
+   "APP_CONTAINERS": "test,hello"
```

Comment:
Is this a backwards compatible change? Can you link istio PR that did this?

Reply:
it should be because it was previously coming in as something like "[\n  test \n, \n hello \n ]".. so it was still a string? 
proxy change:
https://github.com/istio/proxy/pull/2812/
istio change:
https://github.com/istio/istio/pull/22721

Reply:
@kyessenov i have some concerns as well, but it looks like this bit of functionality is (no longer?) working as-is, despite the testing in both repos. we need a path forward here, and I think making the change in proxy and then in istio is the right choice.  

i don't know how to back port this gracefully though.